packages/gizmo/src/Rect.ts

```
@@ -0,0 +1,997 @@
+              // 粒子没有包围盒
+              continue;
+            } else {
+              tempBounds.copyFrom(renderer.bounds);
+              tempBounds.transform(worldInvMatrix);
+              BoundingBox.merge(bounds, tempBounds, bounds);
+            }
+          }
+        }
+      }
+    } else {
+      for (let i = 0, n = entities.length; i < n; i++) {
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)